geomstats/information_geometry/gamma.py

```diff
@@ -356,10 +354,6 @@ def metric_matrix(self, base_point=None):
         mat : array-like, shape=[..., 2, 2]
             Inner-product matrix.
         """
-        if base_point is None:
-            raise ValueError(
-                "A base point must be given to compute the " "metric matrix"
-            )
         base_point = gs.to_ndarray(base_point, to_ndim=2)
```

Comment:
do you think we can avoid this? at first sight looks like if we do `kappa = base_point[..., 0]` we do not need this transformation. What do you think?

Reply:
Yes I agree, I did not really clean everything in this file.